client/layout/index.jsx

```diff
@@ -272,9 +271,6 @@ class Layout extends Component {
 				{ this.props.shouldQueryAllSites && <QuerySites allSites /> }
 				<QueryPreferences />
 				<QuerySiteFeatures siteId={ this.props.siteId } />
-				{ config.isEnabled( 'layout/query-selected-editor' ) && (
-					<QuerySiteSelectedEditor siteId={ this.props.siteId } />
-				) }
```

Comment:
One thing I'm not 100% sure of is if this query component is still needed or not. `isEligibleForGutenframe` is retrieved from the same endpoint, so maybe it's still useful to proactively fetch the data with this component?

Reply:
Thanks for the feedback @pablinos !

> Yes we've seen some issues recently with the editor loading from cold and it needing to wait for API calls to return before being able to continue ...

Given that, I'll plan to refactor `<QuerySiteSelectedEditor>` and the related controller code to focus on `isEligibleForGutenframe` in a separate PR, and then come back to this PR to remove the remaining editor preference code.

Reply:
Sounds good! I feel like it might be possible to include `isEligibleForGutenframe` in another request about the site, to prevent another API request. Maybe that's a future optimisation though.